### ISSUE TITLE
[idea] Render a visual preview of the snippet

### ISSUE THREAD

**Issue by oliviertassinari** [vscodeshift/material-ui-snippets#23]
For instance:

<img width="659" alt="Capture d’écran 2020-10-10 à 16 40 43" src="https://user-images.githubusercontent.com/3165635/95657930-78933a00-0b17-11eb-9a7c-f05e7d08c8a1.png">

<img width="686" alt="Capture d’écran 2020-10-10 à 16 42 24" src="https://user-images.githubusercontent.com/3165635/95657958-9c568000-0b17-11eb-8b7e-319e4901e400.png">

**Comment by jedwards1211** [comment]
Yeah that would be cool.  I'm pretty swamped with other stuff at the moment, but I'd like to implement this eventually!

**Comment by oliviertassinari** [comment]
There is a related issue opened for icons: https://github.com/mui-org/material-ui/issues/19697.

**Comment by arpitBhalla** [comment]
Hey @jedwards1211 I can work on it, if you say

https://github.com/vscodeshift/material-ui-snippets/blob/c9ee8a67122f790fc83e1385b0650968e4efe42d/src/extension.ts#L205

**Comment by jedwards1211** [comment]
@oliviertassinari since a lot of the snippets allow you to select the variant it seems like the previews should show all variants, right?
There should probably be a way for the snippet to declare other defaults to use for previews...

**Comment by jedwards1211** [comment]
I've been slowly working on this.  I finally got all the snippets migrated to a JSX-based declaration system I engineered, so that I'm just a few steps away from actually rendering visual previews.

**Comment by oliviertassinari** [comment]
Should we migrate this issue to https://github.com/vscodeshift/mui-snippets? If I understand it correctly, this repository is meant to be legacy.

**Comment by jedwards1211** [comment]
Yeah that's right.  Since y'all were renaming the package scope from `@material-ui` to `@mui` I thought `mui-snippets` would be a good name for the new repo.  The fact that MUI was intended to encompass more than **M**aterial **UI** totally went over my head.
